docs/user-manual/management.adoc

```diff
@@ -86,6 +86,7 @@ To do this use the `forceFailover()` operation.
 
 Since this method actually stops the server you will probably receive some sort of error depending on which management service you use to call it.
 ====
+* The effective configuration of the broker core can be exported as properties using `exportConfigAsProperties()`. This operation will write a file of the form `java.io.tmpDir/broker_config_as_properties.txt`. The content can provide an entry point for using xref:configuration-index.adoc#broker-properties[brokerProperties].
```

Comment:
The correct value here is `java.io.tmpdir` (i.e. `dir` not `Dir`). It also might be good to clarify that is value is coming from a System property and what that value is expected to be (i.e. `<artemis_instance>/tmp`).

Reply:
Thanks, good catch. Have fixed that and added a Note in the configuration of the tmp dir.